wemake_python_styleguide/visitors/ast/conditions.py

```diff
@@ -111,7 +118,12 @@ def _check_simplifiable_returning_if(self, node: ast.If) -> None:
                 if keywords.is_simple_return(else_body):
                     self.add_violation(SimplifiableReturningIfViolation(node))
                 return
-            body = getattr(node, 'wps_parent').body  # TODO: refactor
+
+            parent = getattr(node, 'wps_parent', None)
+            body = parent.body
+            if self._check_else_body(parent):
```

Comment:
That's strange. We use `self._check_else_body` as a boolean function, but it is not.
It returns `node.body + else_body | None`. But, we don't reuse it. We do the same thing this function does once again with `body = parent.body + parent.orelse`